### ISSUE TITLE
Strange background sdf icon

### ISSUE THREAD

**Issue by usb248** [mapbox/mapbox-gl-js#9876]
**mapbox-gl-js version**: latest

**browser**: last FF

### Steps to Trigger Behavior

and png image fo symbol

### Link to Demonstration

demo : https://bl.ocks.org/malwoodsantoro/8fd46a72e12c8c556a75d0b9f0af1669

public demo
![image](https://user-images.githubusercontent.com/7659927/87645323-971eaa80-c74d-11ea-9c29-99a9592b6d26.png)
Same result in my app
![image](https://user-images.githubusercontent.com/7659927/87645380-b0275b80-c74d-11ea-816f-c098a61dda14.png)

Similar issue : https://github.com/mapbox/mapbox-gl-js/issues/7204 which seems not resolved

### Expected Behavior
not an unexpected background of image
### Actual Behavior
an unexpected background of image

**Comment by arindam1993** [comment]
Yes, this indeed the same problem as #7204 , I think the same suggestion still applies, if you can generate your sdf at a higher resulution it should help.

**Comment by tom-continuumindustries** [comment]
> Yes, this indeed the same problem as #7204 , I think the same suggestion still applies, if you can generate your sdf at a higher resulution it should help.

Increasing the resolution of my SVG's and end images didn't help solve this for me - still having these issues.

**Comment by rreusser** [comment]
@tom-continuumindustries I don't doubt for a second that you're still seeing the issue, but I just want to be certain I understand anything unexpected like platform-dependence. The above linked block shows (in chrome and ff):

<img width="377" alt="Screen Shot 2021-07-29 at 10 17 45 AM" src="https://user-images.githubusercontent.com/572717/127536366-bc49496e-047e-47cb-8329-2d25ebbfaea6.png">

Are you still seeing the outline for that example?
